shared_requirements.txt

```diff
@@ -156,7 +156,7 @@ pyjwt>=1.7.1
 #override azure-storage-file-share azure-core<2.0.0,>=1.10.0
 #override azure-storage-file-datalake azure-core<2.0.0,>=1.10.0
 #override azure-storage-file-datalake msrest>=0.6.21
-#override azure-storage-file-datalake azure-storage-blob<13.0.0,>=12.8.0b1
+#override azure-storage-file-datalake azure-storage-blob<13.0.0,>=12.9.0b1
```

Comment:
I forget if this worked previously, since the blob hasn't been released yet, is the pipeline smart enough to detect unreleased dependency? It seems what we did was releasing blob first then datalake while I cannot remember clearly.

Reply:
I am reverting this change since I removed the ParquetDialect class. But usually yes we would need to do two releases as far as I remember - blob first then datalake. Though that might have changed